AssistantV1.playground/Sources/Helpers.swift

```diff
@@ -16,6 +16,142 @@ public func setupAssistantV1() -> Assistant {
     return assistant
 }
 
+// Find or create a readWrite workspace
+public func getWorkspaceID() -> String {
+
+    let assistant = setupAssistantV1()
```

Comment:
Instead of creating a new instance of assistant in each of these helper methods, it would be better practice to either pass it in as a parameter from the other playground files, or change `setupAssistantV1()` to return a singleton.

Reply:
Returning a singleton sounds like a good change.  We should do this throughout.